### ISSUE TITLE
Default metric/ imperial at power on

### ISSUE THREAD

**Issue by asm308** [clough42/electronic-leadscrew#100]
A little quality of life suggestion. I've modified my setup to start up in metric mode as I have. Metric lathe, but the firmware could determine this based upon whether a metric or imperial leadscrew is defined in the configuration.h

**Comment by J642** [comment]
How did you modified so you start in metric

**Comment by asm308** [comment]
Hi 
If you look in the UserInterface.cop file at around line 64 you will see a line that says
this->metric=false
Change false true and recompile.
If I was doing it for a general solution I’d have set it depending on whether LEADSCREW_HMM was defined, but being a bit lazy

**Comment by clough42** [comment]
This is on my list of features to add.  Though rather than making it configurable, I was planning on making the ELS remember its settings in the EEPROM flash so if you set it to metric, it'll still be in metric next time you turn it on.

**Comment by asm308** [comment]
Yeah that would be a better solution. I’ll keep an eye out for it.  Thanks
